nvme_gw.config

```diff
@@ -25,8 +25,7 @@ client_cert = ./client.crt
 
 [spdk]
 
-spdk_server_addr = /var/tmp/spdk.sock
-spdk_port = 5260
+rpc_socket = /var/tmp/spdk.sock
```

Comment:
I'd prefer this to be called spdk_socket

Reply:
As it is stated in the commit log message, I don't see much sens to use `spdk_` prefix for parameters in `[spdk]` section.